apps/web/configuration/localeConfig.ts

```diff
@@ -0,0 +1,47 @@
+import { LocaleObject } from '@nuxtjs/i18n';
+import path from 'node:path';
+import * as fs from 'node:fs';
+
+export const getLocaleObject = () => {
+  let localeObject: LocaleObject[] = [
+    { code: 'en', file: 'en.json' },
+    { code: 'de', file: 'de.json' },
+  ];
+
+  if (process.env.LANGUAGELIST !== undefined) {
+    localeObject = [];
```

Comment:
```suggestion
  let localeObject: LocaleObject[] = [];

  if (process.env.LANGUAGELIST !== undefined) {
```
**suggestion**: I don't think we need to instantiate the array with `en` and `de`. We take care of the fallback at the end, after all.

Reply:
What about the case where the LANGUAGELIST is missing from env?